inc/admin/ui/modules/cdn.php

```diff
@@ -137,7 +141,7 @@
 			'type'         => 'textarea',
 			'label_for'    => 'cdn_reject_files',
 			'label_screen' => __( 'Exclude files:', 'rocket' ),
-			'placeholder'  => "e.g. /wp-content/themes/some-theme/style.css\ne.g. /wp-content/plugins/some-plugin/(.*).css"
+			'placeholder'  => "/wp-content/themes/some-theme/style.css\n/wp-content/plugins/some-plugin/(.*).css"
```

Comment:
Done in https://github.com/wp-media/wp-rocket/pull/418/commits/6696b048b0799dbd6882b820ee493ae18814fa38